dynamic-config/config-kie/src/main/java/org/apache/servicecomb/config/kie/client/KieClient.java

```diff
@@ -127,29 +99,38 @@ public void destroy() {
     @Override
     public void run() {
       try {
-        refreshConfig();
+        CountDownLatch latch = new CountDownLatch(1);
```

Comment:
why this place use CountDownLatch ?

Reply:
1. make refreshConfig synchronouse. In pull mode, when next pull happends, if this is asynchronouse,  it is possible two pulls in the same time. Alghough this is rarely happen
2. In long polling mode, to prevent from refreshConfig caught some error, and next long polling not happen. This older implementations do not handle execpetion properly and long polling will exit when timeout happens.